README.md

```diff
@@ -1,2 +1,114 @@
 # covtracer-action
```

Comment:
I also found "CovtracerCheck action" name in the text and wanted to double check the final name of this GH action.

Reply:
changed